frontend/mr-admin-flate/src/components/avtaler/AvtaleSchema.ts

```diff
@@ -18,6 +18,12 @@ export const AvtaleSchema = z.object({
     .max(9, "Du må velge en leverandør")
     .regex(/^\d+$/, "Leverandør må være et nummer"),
   leverandorUnderenheter: z.string().array(),
+  leverandorKontaktperson: z
+    .object({
+      navn: z.string().optional(),
+      telefon: z.string().optional(),
+      epost: z.string().optional(),
```

Comment:
Mener du kan bruke `z.string().email()` for å validere at det er en gyldig epost-adresse https://zod.dev/?id=strings

Reply:
Nice!

Reply:
I den nye komponenten kunne jeg ikke bruke form (fordi form inni form ikke var lov). Så jeg droppa denne